Source/drlg_l3.cpp

```diff
@@ -1970,7 +1970,7 @@ void FenceDoorFix()
 
 static void DRLG_L3Wood()
 {
-	int i, j, x, y, xx, yy, rt, rp, x1, y1, x2, y2;
+	volatile int i, j, x, y, xx, yy, rt, rp, x1, y1, x2, y2;

```

Comment:
that's not a part of discord features ;)

Reply:
Yeah sorry, can't compile without it, I'll remove it later.

Reply:
:D god bless vs2019

Reply:
Maybe, it's an Easter egg as dedicated to M_LazHelp